src/main/java/com/codeborne/selenide/SelenideElement.java

```diff
@@ -954,6 +954,21 @@
   @Nonnull
   WebElement getWrappedElement();
 
+  /**
+   * Click the element using {@link com.codeborne.selenide.ClickOptions}
+   *
+   * @see com.codeborne.selenide.commands.Click
+   */
+  void click(ClickOptions clickOption);
+
+  /**
+   * Click the element using {@link com.codeborne.selenide.ClickOptions}
+   * with a relative offset from the center of the element
+   *
+   * @see com.codeborne.selenide.commands.Click
+   */
+  void click(ClickOptions clickOption, int offsetX, int offsetY);
```

Comment:
Ideally, we would like to avoid too many overloaded methods in class `Selenide`. 
Could we only have method `click(ClickOptions clickOption)`, and move parameters `int offsetX, int offsetY` into `ClickOptions` class?

Reply:
Done. Changed to `click(ClickParams);`